src/api/controller/rootAdmin.js

```diff
@@ -131,8 +131,12 @@ const deleteTenant = async ctx => {
         ctx.status = 403;
         ctx.body = { error: `Invalid name: "${name}"` };
     } else {
-        deleteWorkerQueue(tenantExists.name);
+        deleteWorkerQueue(tenantExists.name).then();
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏What?

I don't get the utility of this `.then()`.
OK it shows that `deleteWorkerQueue` is asynchronous.

I just hope it's not just an unachieved development.

Reply:
This is just to remove warring, this function is call, be we don't use the result